HTWDD/Main Categories/General/AppDelegate.swift

```diff
@@ -43,6 +43,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         Tracker.track(.open)
     }
 
+    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
+        
+        let viewController = url.host?.removingPercentEncoding
+        
+        if(viewController == "schedule") {
+            let tababarController = self.window!.rootViewController as! UITabBarController
```

Comment:
Please don't use implicitly unwrapped optionals if it's not needed. You could also just write that using `?` and `as?`.
Also: we use the coordinator pattern, so this should not propagated to the view controller but to the coordinator instead. I know that this might be a bit over-complicated, so if you don't feel like changing that - ping me and I do it :)

- [ ] avoid implicitly unwrapped optionals
- [ ] forward action to coordinator instead of view controller

Reply:
Yeah, I saw the coordinator and will have a deeper look into it.